ui-tests/src/test/java/org/openmrs/reference/ComeBackToApplicationAfterLogoutTest.java

```diff
@@ -11,10 +11,13 @@
 package org.openmrs.reference;
 
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
 
 public class ComeBackToApplicationAfterLogoutTest extends ReferenceApplicationTestBase {
 
     @Test
+    @Category(BuildTests.class)
```

Comment:
Adding `@Category(BuildTests.class) `annotation to align with the `method` test convention.